doc/source/library_reference/library/SpotAnalysis/base_classes.rst

```diff
@@ -0,0 +1,52 @@
+SpotAnalysis
+============
+
+.. currentmodule:: opencsp.common.lib.cv.SpotAnalysis
+
+.. automodule:: opencsp.common.lib.cv.SpotAnalysis
+   :members:
+   :special-members: __init__
+   :undoc-members:
+   :show-inheritance:
+   :member-order: groupwise
+
+Data Classes
+============
+
+These are the book keeping classes that enable the spot analysis pipeline.
+
+.. currentmodule:: opencsp.common.lib.cv.spot_analysis
+
+.. automodule:: opencsp.common.lib.cv.spot_analysis.SpotAnalysisOperable
+   :members:
+   :undoc-members:
+   :show-inheritance:
+   :member-order: groupwise
+
```

Comment:
```suggestion

.. automodule:: opencsp.common.lib.cv.spot_analysis.SpotAnalysisOperablesStream
   :members:
   :undoc-members:
   :show-inheritance:
   :member-order: groupwise

```

Reply:
I would be inclined to include all non-protected and non-private entities in the documentation

Reply:
Todo: indicate that this class is not meant for use by users, and most of the time not for use by developers. It is necessary for how the SpotAnalysis class works but only users who really want a deep dive should take a look. Maybe _SpotAnalysisOperablesStream or __SpotAnalysisOperablesStream?

Reply:
Done!